tests/queries/0_stateless/02346_text_index_direct_read.sql

```diff
@@ -0,0 +1,50 @@
+-- Tags: no-fasttest
+-- no-fasttest: It can be slow
```

Comment:
Are you sure? This test looks like it runs in a couple of seconds. The only slow operation may the logs flushing.

If other similar tests which check `system.text_log` also don't run in FastTests, then we can exclude this test too, otherwise it will be nicer to remove l. 1-2.

Reply:
Fixed. Previously it needed ~20 seconds to run, but after [6eddda7](https://github.com/ClickHouse/ClickHouse/pull/87955/commits/6eddda7a9c0e19d35c85e0ebcc0f8b2c50edb17f) it only takes ~2 seconds.